packages/dotcom-middleware-navigation/src/handleEdition.ts

```diff
@@ -11,8 +11,8 @@ export default (request: Request, response: Response): string => {
   // <https://github.com/Financial-Times/next-router/blob/master/server/middleware/editions.js>
   let currentEdition = request.get('FT-Edition') || defaultEdition
 
-  if (request.query.edition && isEdition(request.query.edition)) {
-    currentEdition = request.query.edition
+  if (request.query.edition && isEdition(request.query.edition as string)) {
```

Comment:
How about this:

```ts
if (typeof request.query.edition === 'string' && isEdition(request.query.edition))
```

TS should be aware of the `typeof` check 🤞

Reply:
Success! TypeScript is appeased.

Change added in [this commit](https://github.com/Financial-Times/dotcom-page-kit/pull/796/commits/76c72d9feef4540ca2ff986d281b9cb5b2926354).